build/commands/lib/config.js

```diff
@@ -1618,9 +222,8 @@ Config.prototype.buildArgs = function () {
     enable_hangout_services_extension: this.enable_hangout_services_extension,
     enable_cdm_host_verification: this.enableCDMHostVerification(),
     skip_signing: !this.shouldSign(),
-    sparkle_eddsa_public_key: this.sparkleEdDSAPublicKey,
-    sparkle_eddsa_private_key: this.sparkleEdDSAPrivateKey,
```

Comment:
@mihaiplesa Why did you delete private key? We need it for eddsa signing for patch and dmg file.

Reply:
I reverted this change by https://github.com/brave/brave-core/pull/6846/commits/c61bb41e9430432f908436fbdec0f4978055b3f7
I assumed you deleted this accidently.